scripts/pip

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -e
+# Use docker + pip-compile-multi to pin scriptworker python requirements
+TOP_DIR="$(dirname "$( cd "$(dirname "$0")" && pwd -P)")"
+PIP_DOCKER_TAG="${PIP_DOCKER_TAG:-pip}"
```

Comment:
I think this should be something like `mozillareleases/scriptworker-pip-compile:${PY_DOT_VERSION}`, since the namespace is global to the machine?

Reply:
Makes sense. I'm going to figure out how to push to mozillareleases.